controllers/articles.controllers.js

```diff
@@ -9,3 +12,15 @@ exports.getArticleByID = (req, res, next) => {
       next(err);
     });
 };
+
+exports.patchVotesOfArticleByID = (req, res, next) => {
+  const article_id = req.params.article_id;
+  const newVote = req.body.inc_votes;
```

Comment:
Picky picky but you've assigned `req.body.inc_votes` to a variable that follows camelCase standard. However your article_id doesn't do the same. Just keep consistent with your decisions and either make article_id camel case or newVote not.